package.json

```diff
@@ -94,8 +94,10 @@
     "cypress-file-upload": "5.0.2",
     "cypress-localstorage-commands": "1.4.5",
     "env-cmd": "10.1.0",
+    "eslint": "^7.31.0",
```

Comment:
Can we pin these two repos?

Reply:
It's better not pin them, as it's actually dependent on `react-scripts`'s same dependency. I've tried and it failed.